### PR TITLE
Touch up readme tests

### PR DESCRIPTION
**Pull request by bvanderlaan**
The tests under `test/readme` would not mark a test failed if async operations returned an error.
The tests was just logging the error but still marking the test success.

This PR refactors these tests so that they properly report test failures.

This PR is dependent on https://github.com/Flux159/cassie-odm/pull/11
It uses the `cassieconnect` module defined in the above PR.

I ran all the tests against Node v4, v6, and 8 they all still pass.
I tried mutating the code to cause errors to ensure the tests fail as expected.

### CODE REVIEW

File: test/readme/client_connections_test.js [hunk 1]
```diff
@@ -1,21 +1,18 @@
-var assert = require('assert');
+'use strict';
 
+var config = require('../cassieconnect').connectOptions;
 var cassie = require('../../lib/cassie');
 
-describe('Client Connections', function() {
+describe('Client Connections and raw queries', function() {
+    it('should not fail', function (done) {
+        // var cassie = require('cassie-odm');
```
Comment (Flux159) [inline test/readme/client_connections_test.js]: Can remove these comments

File: test/readme/getting_started_test.js [hunk 2]
```diff
@@ -1,31 +1,28 @@
...
+var cassie = require('../../lib/cassie');
 
+describe('Getting Started', function() {
     it('should not fail', function (done) {
-
-//            var cassie = require('cassie-odm'); //One change from the README
-        var cassie = require('../../lib/cassie');
-        var config = {keyspace: "CassieTest", hosts: ["127.0.0.1:9042"]};
-        cassie.connect(config);
+        // var cassie = require('cassie-odm');
```
Comment (Flux159) [inline test/readme/getting_started_test.js]: Same, should probably remove these comments

**Review by Flux159 — APPROVED** [review]
Just some cleanup around removing comments

File: test/readme/getting_started_test.js (hunk 2 above)
Comment (bvanderlaan) [reply test/readme/getting_started_test.js]: will do 👍

**Pull request merged**